package.json

```diff
@@ -30,24 +30,66 @@
     "tamper": "latest"
   },
   "devDependencies": {
+    "amdefine": "latest",
+    "browserify": "latest",
+    "browserify-shim": "latest",
     "chai": "latest",
     "grunt": "latest",
     "mocha": "latest",
+    "grunt-browserify": "latest",
     "grunt-bowercopy": "latest",
     "grunt-cli": "latest",
     "grunt-contrib-concat": "latest",
     "grunt-contrib-copy": "latest",
     "grunt-contrib-imagemin": "latest",
     "grunt-contrib-less": "latest",
-    "grunt-contrib-requirejs": "latest",
     "grunt-contrib-uglify": "latest",
     "grunt-contrib-watch": "latest",
     "grunt-express-server": "latest",
     "grunt-mocha-test": "latest",
     "grunt-node-inspector": "latest",
-    "grunt-replace": "latest",
     "grunt-sed": "latest",
     "node-phantom-simple": "latest",
-    "phantomjs": "latest"
+    "phantomjs": "latest",
+    "watchify": "latest"
+  },
+  "browser": {
+    "codescan": "./content/src/codescan.js",
+    "controller": "./content/src/controller.js",
+    "debug": "./content/src/debug.js",
+    "draw-protractor": "./content/src/draw-protractor.js",
+    "droplet": "./content/lib/droplet.js",
+    "FontLoader": "./content/src/FontLoader.js",
+    "filetype": "./content/src/filetype.js",
+    "gadget": "./content/src/gadget.js",
+    "guide": "./content/src/guide.js",
+    "iced-coffee-script": "./content/lib/iced-coffee-script.js",
+    "jquery": "./content/lib/jquery.js",
+    "palette": "./content/src/palette.js",
+    "pencil-tracer": "./content/lib/pencil-tracer.js",
+    "see": "./content/lib/see.js",
+    "seedrandom": "./content/lib/seedrandom.js",
+    "sourcemap": "./content/lib/sourcemap/source-map-consumer.js",
```

Comment:
OK - let's go ahead and switch to using npm for these four dependencies for now.  Also, let's switch droplet - @dabbler0, can you "npm publish" droplet?